WebCryptoAPI/derive_bits_keys/derived_bits_length_testcases.js

```diff
@@ -0,0 +1,18 @@
+var testCases = {
+    "HKDF": [
+        {length: 256, expected: algorithms["HKDF"].derivation},
+        {length: 0, expected: emptyArray},
+    ],
+    "PBKDF2": [
+        {length: 256, expected: algorithms["PBKDF2"].derivation},
+        {length: 0, expected: undefined},
```

Comment:
This seems like this 0-length inconsistency with HKDF and PBKDF2 should be fixed at the same time. Pick one.

Reply:
Do you mean those test cases are redundant ? I think there is still value on being exhaustive on the algorithms included in the subtests. Additionally, the undefined expected value means that the test will check that an OperationError exception is thrown, and not a different one.

Reply:
I mean that the tests produce a different outcome, when the behaviour we're requesting is essentially the same ("generate 0 bytes from your KDF please").

Reply:
Yeah, indeed. I was confused by the fact that in the TR version we were not considering non-multiple of 8 cases as OperatonError. It's true that we have fixed this already in the last spec draft. 

I'll fix this by using the 'undefined' value for the expected result, so we assert this case throws OperationError for both algorithms.